hq/conf/riff-raff.yaml

```diff
@@ -14,7 +14,7 @@ deployments:
     parameters:
       prependStackToCloudFormationStackName: false
       cloudFormationStackName: security-hq
-      templatePath: cfn.json
+      templatePath: security-hq.template.json
```

Comment:
`riffRaffArtifactResources` can no longer rename the CDK generated file to `cfn.json`, so let's just use the CDK generated filename.